components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/store/JDBCIdentityDataStore.java

```diff
@@ -266,6 +273,62 @@ public void remove(String userName, UserStoreManager userStoreManager) throws Id
         }
     }
 
+    @Override
+    public List<String> list(String claimUri, String claimValue,
+                             org.wso2.carbon.user.core.UserStoreManager userStoreManager) throws IdentityException {
+
+        List<String> userNames = new ArrayList<>();
+
+        // This is to support LDAP like queries, so this will provide support for only leading or trailing '*'
+        // filters. if the query has multiple '*' s the filter will ignore all.
+        if (claimValue.contains(QUERY_FILTER_STRING_ANY)) {
+            if ((claimValue.startsWith(QUERY_FILTER_STRING_ANY) &&
+                    !claimValue.substring(1).contains(QUERY_FILTER_STRING_ANY)) ||
+                    claimValue.endsWith(QUERY_FILTER_STRING_ANY) &&
+                            !claimValue.substring(0, claimValue.length() - 1).contains(QUERY_FILTER_STRING_ANY) &&
+                            claimValue.charAt(claimValue.length() - 2) != SQL_FILTER_CHAR_ESCAPE) {
+                claimValue = claimValue.replace(QUERY_FILTER_STRING_ANY, SQL_FILTER_STRING_ANY);
```

Comment:
Shall we add a debug log indicating which final claimValue being searched ?

Reply:
Added all at once
https://github.com/wso2-extensions/identity-governance/pull/170/files#diff-eb8649c3eedd56d1e6d03a87058c6574R320